### ISSUE TITLE
[Components and pattern standards] Design component or pattern in use isn't mobile-friendly. (04.09.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#43141]
#### VFS team name
Public Websites

#### VFS product name
Sitwwide Header

#### VFS feature name
IE11 Deprecation Alert

#### Point of Contact/Reviewers
Brian DeConinck (@briandeconinck) - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Design component or pattern in use isn't mobile-friendly.

### Issue Details
At zoom greater than 200%, the deprecation alert is not visible. This impacts low-vision users who keep their browser at high zoom levels to improve readability. Per [WCAG 1.4.10](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html), we support up to 400% zoom.

For users who (1) use IE 11 and (2) keep their browser at zoom levels higher than 200%, this is information that's just missing. Since the deprecation alert is advisory only and there's no immediate loss of functionality, I'm marking this with label `508-defect-2` and not categorizing it as a launch-blocker. But please look at resolving soon after launch.

### Link, screenshot or steps to recreate
1. Navigate to the review instance in IE11 or a browser set to the IE11 user agent string.
2. Resize your browser window to a starting browser viewport of 1280px width.
3. Use `Ctrl` + `+` to change your zoom level.
4. Note that the deprecation alert is visible up to 200%, but disappears at zoom levels beyond that.

### VA.gov Experience Standard
[Category Number 04, Issue Number 09](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.4.10 AA

---

### Platform Recommendation
It looks like the deprecation alert is only present in the `#legacy-header` div for the tablet/desktop header, and not included in the mobile instance of the header. That makes sense since IE11 isn't available on mobile devices, but that assumption breaks down for high zoom users. Browser zoom triggers responsive breakpoints, and low-vision users are frequently viewing the "mobile" layout of a page on a laptop or desktop.

Recommendation: Include the deprecation alert in the mobile layout too. I don't think much needs to happen to the banner itself to make it high-zoom friendly, it just needs to render.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by wesrowe** [comment]
@briandeconinck, thanks for the clear steps to reproduce. You educated us on a use case we weren't aware of – the cause is that we implemented the banner in the desktop header only, thinking ie11 is unlikely to be used in a manner that uses the separate mobile header (which triggers at narrow screen widths). We're looking into adding this asap.

**Comment by wesrowe** [comment]
Update: Claudio reminded me that we added the banner to the mobile header as well. The testing methodology caused it not to show in the review instance. If the mobile/zoomed version of the banner fails to show in production, we will address immediately post release.

**Comment by laflannery** [comment]
@briandeconinck I am able to see this zoomed in higher than 200% (up to 400%) now.  Would you be able to retest this and if you are still seeing the issue?

Screenshot at 250%:
![image](https://user-images.githubusercontent.com/106678594/192364655-af2796f8-2c6f-4208-a24f-adcc180e2733.png)
